public/js/app.js

```diff
@@ -0,0 +1,202 @@
+const menu = document.querySelectorAll('.item-menu');
+const h1 = document.querySelector('h1');
+const contain = document.querySelector('.contain');
+const currentDate = document.querySelector('.current-date p');
+let books = [];
+
+/* eslint-disable no-unused-vars */
+const menuActive = (menu, active) => {
+  for (let i = 0; i < menu.length; i += 1) {
+    menu[i].classList.remove('active');
+  }
+  active.classList.add('active');
+};
+const generateTable = () => {
+  contain.innerHTML = '';
+  const table = document.createElement('table');
+  // header
+  const tr = document.createElement('tr');
+  const th1 = document.createElement('th');
+  th1.textContent = '#';
+  const th2 = document.createElement('th');
+  th2.textContent = 'Title';
+  const th3 = document.createElement('th');
+  th3.textContent = 'Author';
+  const th4 = document.createElement('th');
+  th4.textContent = 'Created';
+  const th5 = document.createElement('th');
+  th5.textContent = 'Actions';
+  tr.appendChild(th1);
+  tr.appendChild(th2);
+  tr.appendChild(th3);
+  tr.appendChild(th4);
+  tr.appendChild(th5);
+  table.appendChild(tr);
+  // body
+  books.forEach((book, i) => {
+    const tr1 = document.createElement('tr');
+    const th11 = document.createElement('th');
+    th11.textContent = i + 1;
+    const th12 = document.createElement('td');
+    th12.textContent = book.title;
+    const th13 = document.createElement('td');
+    th13.textContent = book.author;
+    const th14 = document.createElement('td');
+    th14.textContent = book.created_at;
+    const th15 = document.createElement('td');
+    th15.innerHTML = `<a href=""><i class="fa fa-search-plus" aria-hidden="true"></i></a> | <a href="#" onclick="removeBook(${
+      i
+    })"><i class="fa fa-trash-o" aria-hidden="true"></i></a>`;
+    tr1.appendChild(th11);
+    tr1.appendChild(th12);
+    tr1.appendChild(th13);
+    tr1.appendChild(th14);
+    tr1.appendChild(th15);
+    table.appendChild(tr1);
+  });
+  contain.appendChild(table);
+};
+const generateAddForm = () => {
+  const parent = document.createElement('form');
+  parent.setAttribute('action', '');
+  parent.setAttribute('method', 'POST');
+  parent.setAttribute('onsubmit', 'return validateForm()');
+  parent.setAttribute('id', 'add_book');
+  const l1 = document.createElement('fieldset');
+  l1.innerHTML = '<legend><label for="title">Title</label></legend>';
+  parent.appendChild(l1);
+  // input title
+  const input = document.createElement('input');
+  input.setAttribute('type', 'text');
+  input.setAttribute('name', 'title');
+  input.setAttribute('id', 'title');
+  input.setAttribute('placeholder', 'Write the title');
+  input.setAttribute('value', 'Book 1');
```

Comment:
- [ ] It would be better to make your input value empty so users do not need to clear Book 1 before typing the Book name or Title